recipe/meta.yaml

```diff
@@ -13,7 +13,6 @@ source:
 
 build:
   number: 0
-  skip: True  # [py2k]
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
```

Comment:
Not this exact line, but on line 35, could you change:

` - enum34 >=1.1.6 # [py<34]`

to:

`- enum34 >=1.1.6  # [py<34]`

?

Note - there's just one extra space in there.

Reply:
There is a PR in conda-smithy that will output the offending line to make situations like these easier to fix in the near future.

Reply:
Thanks @ocefpaf!

Reply:
Ping @kpalin, please see comment above re: `enum34`.

Reply:
Sorry. I'll plea "inexperience with 'git commit' syntax" It's coming now.